common/2021.04.06/amymeme_10830.py

```diff
@@ -0,0 +1,41 @@
+# https://www.acmicpc.net/problem/10830
+# 백준 10830 - 행렬 제곱
+
+import sys
+
+
+def get_elem(sub_list1, sub_list2):
+    return sum([a * b for a, b in zip(sub_list1, sub_list2)]) % 1000
```

Comment:
zip([1,2],[1,3]) -> [[1,1], [2,3]]
[a * b for a, b in [[1,1], [2,3]] -> [1, 6]
sum[1,6] -> 7